pkgs/servers/home-assistant/default.nix

```diff
@@ -84,7 +89,13 @@ in with py.pkgs; buildPythonApplication rec {
   };
 
   # leave this in, so users don't have to constantly update their downstream patch handling
-  patches = [];
+  patches = [
+    (fetchpatch {
+      #  Add timeouts in stream tests to prevent possible hangs
+      url = "https://github.com/home-assistant/core/pull/47638.patch";
```

Comment:
```suggestion
      url = "https://github.com/home-assistant/core/commit/740edbb653a16f01c0cf9ed309492663266e9c6a.patch";
```

Reply:
Yep, just using our ofBorg infrastructure to test-drive the changes. Will adapt that as soon as the final commit is merged.